src/components/config/validate-config.ts

```diff
@@ -22,29 +22,27 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-import yaml from 'js-yaml'
-import fs from 'fs'
 import Ajv from 'ajv'
 import { Validation } from '../../interfaces/validation'
 import mySchema from '../../monika-config-schema.json'
+import { Config } from '../../interfaces/config'
 
 const ajv = new Ajv()
 
-// validate the config file used by monika
-export function validateConfigFile(filename: string): Validation {
+// validate the config file  loaded by monika against a JSON Schema
+export function validateConfigFile(config: Config): Validation {
```

Comment:
maybe we should rename this function since it doesn't actually validate a config file. To make it clearer, how about `validateConfigWithSchema`